webapp/portlet/src/main/webapp/vue-apps/vertical-menu/components/VerticalMenuContent.vue

```diff
@@ -0,0 +1,52 @@
+<!--
+  This file is part of the Meeds project (https://meeds.io/).
+  Copyright (C) 2023 Meeds Association
+  contact@meeds.io
+  This program is free software; you can redistribute it and/or
+  modify it under the terms of the GNU Lesser General Public
+  License as published by the Free Software Foundation; either
+  version 3 of the License, or (at your option) any later version.
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+  Lesser General Public License for more details.
+  You should have received a copy of the GNU Lesser General Public License
+  along with this program; if not, write to the Free Software Foundation,
+  Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+-->
+<template>
+  <v-app id="verticalMenuSiteDetails">
+    <v-card class="elevation-0" :class="extraClass">
+      <site-details
+        v-if="site"
+        display-sequentially
+        :site="site"
+        :extra-class="' px-0 py-0 '" />
+    </v-card>
+  </v-app>
+</template>
+<script>
+export default {
+  props: {
+    extraClass: {
+      type: String,
+      default: ' '
+    }
+  },
+  data: () => ({
+    site: null,
+    siteId: eXo.env.portal.siteId,
+  }),
+  created() {
+    this.retrieveCurrentSite();
+  },
+  methods: {
+    retrieveCurrentSite(){
+      return this.$siteService.getSiteById(this.siteId, true, true, eXo.env.portal.language, ['displayed', 'temporal'], true, true)
+        .then(site => {
+          this.site = site;
+        });
```

Comment:
Add a **loading effect**. Even in Automatic Tests, this will help to know when the DOM elements are loaded.
In addition, it offers a coherent UX with other components of the platform.

```suggestion
      this.loading = true;
      return this.$siteService.getSiteById(this.siteId, {
        expandNavigations: true,
        excludeEmptyNavigationSites: true,
        lang: eXo.env.portal.language,
        visibility: ['displayed', 'temporal'],
        excludeGroupNodesWithoutPageChildNodes: true,
        temporalCheck: true,
      })
        .then(site => this.site = site)
        .finally(() => this.loading = false);
```